src/server.rs

```diff
@@ -392,7 +422,10 @@ mod test {
     }
 
     fn device_id() -> String {
-        thread_rng().gen_ascii_chars().take(8).collect()
+        iter::repeat(())
```

Comment:
that sample_iter I mentioned in the last PR's: `thread_rng().sample_iter(&distributions::Alphanumeric).take(8).collect()`